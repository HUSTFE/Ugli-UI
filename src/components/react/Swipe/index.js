import React, { Component } from 'react'
// import PropTypes from 'prop-types'

// import styles from '@style/Swipe/index.sass'

class Swipe extends Component {
  static propTypes = {
    // size: PropTypes.oneOfType([
    //   PropTypes.string,
    //   PropTypes.number,
    // ]),
  }
  static defaultProps = {
    // size: '16',
  }
  render() {
    return (<div>This is Swipe component.</div>)
  }
}

export default Swipe
