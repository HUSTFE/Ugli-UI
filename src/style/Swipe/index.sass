.swiper-container {
  overflow: hidden;
  > .swiper-wrapper {
    
  }
}