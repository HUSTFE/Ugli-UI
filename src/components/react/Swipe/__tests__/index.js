import React from 'react'
import renderer from 'react-test-renderer'
import Swipe from '../index'

describe('Swipe', () => {
  it('Swipe component render without crash', () => {
    const component = renderer.create(<Swipe />)
    const json = component.toJSON()
    expect(json).toMatchSnapshot()
  })
})
