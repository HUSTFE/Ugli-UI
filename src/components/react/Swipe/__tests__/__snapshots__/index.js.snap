// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Swipe snake spinner render without crash 1`] = `
<div>
  This is Swipe component.
</div>
`;
